docker-compose/config/pm2-dev.json

```diff
@@ -3,7 +3,7 @@
     "name"         : "KuzzleServer",
     "script"       : "bin/kuzzle",
     "args"         : ["start"],
-    "watch"        : ["lib/**/*.js", "default.config.js", "plugins/enabled/**.js"],
+    "watch"        : ["lib/**/*.js", "default.config.js", "plugins/enabled/**/**.js"],
```

Comment:
nitpicking: shouldn't it be, instead `"plugins/enabled/**/*.js"` (with a single '*' at the end) ?